.github/workflows/tests.yml

```diff
@@ -0,0 +1,73 @@
+name: tests
+
+on: push
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  rust:
+    name: rust build
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+
+      - name: Build
+        working-directory: p2panda-rs
+        run: cargo build --verbose
+
+      - name: Run tests
+        working-directory: p2panda-rs
+        run: cargo test --verbose
+  js:
+    name: js build
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Read node version from .nvmrc
+        working-directory: p2panda-js
```

Comment:
Do you know if its possible to set the working directory only once somewhere? I could find this discussion here: https://github.community/t/use-working-directory-for-entire-job/16747 and it seems to be a problem, but also in the end someone solved it like this?

```
defaults:
  run:
    working-directory: ./your_working_dir
jobs:
  ...
```

Not sure if it will help in our case, its more of a question :thinking:

Reply:
I looked around a bit and tried cd but that didn't work :( i can try the syntax you linked, yes!